### PR TITLE
Update 1password to 7.0

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/1password.rb [hunk 1]
```diff
@@ -15,23 +15,33 @@
     url "https://cache.agilebits.com/dist/1P/mac4/1Password-#{version}.zip"
 
     app "1Password #{version.major}.app"
-  else
+  elsif MacOS.version <= :sierra
```
Comment (vitorgalvao) [inline Casks/1password.rb]: This implies this version only works on Sierra and below, which I don’t think is the case.

Comment (cpmsmith) [reply Casks/1password.rb]: According to AgileBits, [v7 supports 10.12 onward](https://support.1password.com/upgrade-mac/#get-help), so this should say El Capitan.

**Comment by brianmorton** [comment]
https://github.com/caskroom/homebrew-cask/pull/47470

File: Casks/1password.rb (hunk 1 above)
Comment (core-code) [reply Casks/1password.rb]: > This implies this version only works on Sierra and below, which I don’t think is the case.
 
i think you mis-read the diff. version 7 of 1password only runs on 10.12+ so older systems still have to install the old version 6

> This needs to be part of only the conditional of version 7.

fixed thanks

**Review by commitay — CHANGES_REQUESTED** [review]
Version 7 is a paid upgrade, I think this Cask should contain version 7 only and 6 should be added to versions.

**Comment by commitay** [comment]
Also, there is a zip for version 7.

https://cache.agilebits.com/dist/1P/mac7/1Password-7.0.zip

**Comment by core-code** [comment]
the zip file wasn't there when i checked earlier today. 

i can change the cask to contain only version 7 if you want.

**Review by commitay — APPROVED** [review]

File: Casks/1password.rb [hunk 2]
```diff
@@ -1,36 +1,18 @@
...
-
-    # cache.agilebits.com/dist/1P/mac4 was verified as official when first introduced to the cask
-    url "https://cache.agilebits.com/dist/1P/mac4/1Password-#{version}.zip"
-
-    app "1Password #{version.major}.app"
-  end
-
-  appcast 'https://app-updates.agilebits.com/product_history/OPM4',
-          checkpoint: 'eb8b60476f30a53f496a84e809b948faf81ea9f9ab442f4c923e48720e5a1f2a'
+  # 1password.com was verified as official when first introduced to the cask
```
Comment (commitay) [inline Casks/1password.rb]: The comment is required, same as https://github.com/caskroom/homebrew-versions/pull/5451

**Comment by commitay** [comment]
PR for `1password6` https://github.com/caskroom/homebrew-versions/pull/5720

**Review by karlbright — APPROVED** [review]

**Comment by vitorgalvao** [comment]
> the zip file wasn't there when i checked earlier today.

Can confirm. I also checked and thought they had given rid of it for good.

**Pull request merged**